### PR TITLE
Avatar overlap

### PR DESCRIPTION
**Pull request by janwillemtulp**
For now this could be ok, since the avatars move a bit closer to the globe once you zoom in. All avatars are clickable then. But with more and closer avatars, this can become an issue again.

--- thoughts for future solutions ---

The suggestion of applying a Force Directed Layout could work, but the simulation would have to restart on every zoom interaction, because the relative size of the avatars changes with the zoom level (and thus the node repulsion).

The best alternative (not yet tested) is to calculate the positions (without overlap) at minimum and maximum zoom levels, and then for each avatar provide a local scale that allows to map the zoom level to a translation in position.

### CODE REVIEW

File: components/globe-component/avatar.js [hunk 1]
```diff
@@ -17,6 +12,16 @@ class Avatar {
     this.mesh = new THREE.Object3D()
   }
 
+  /**
+   * Updates the distance from the globe based on the camera z position
+   * @param {int} cameraZ camera Z position
+   */
+  updateDistanceFromGlobe (cameraZ) {
+    [this.mesh.children[0]].forEach(avatar => {
+      console.log(avatar.position.z)
```
Comment (jurgenbelien) [inline components/globe-component/avatar.js]: This only logs the position? then you can probably remove it, right?

File: components/globe-component/globe-component.js [hunk 2]
```diff
@@ -277,6 +299,13 @@ export default {
       const dirLight = new THREE.DirectionalLight(0xffaa66, 4.2)
       dirLight.position.set(15, 13, 15)
 
+      // const that = this
+
+      // this.controls.addEventListener('change', function () {
+      //   console.log(this.object.position.z)
+      //   that.avatar.updateDistanceFromGlobe(this.object.position.z)
+      // }, false)
```
Comment (jurgenbelien) [inline components/globe-component/globe-component.js]: You can remove this inactive code I guess?

File: components/globe-component/avatar.js (hunk 1 above)
Comment (janwillemtulp) [reply components/globe-component/avatar.js]: yes... will remove that

File: components/globe-component/globe-component.js (hunk 2 above)
Comment (janwillemtulp) [reply components/globe-component/globe-component.js]: yes!

**Pull request merged**